great_expectations_cloud/agent/agent.py

```diff
@@ -144,7 +144,7 @@ def _listen(self) -> None:
             print("Received request to shutdown.")
         except (SubscriberError, ClientError):
             print("Connection to GX Cloud has encountered an error.")
-        except AuthenticationError:
+        except (AuthenticationError, ProbableAuthenticationError):
```

Comment:
I think we need to also add `ProbableAuthenticationError` to the `@retry` decorator above on line 126 to enable the retry: https://tenacity.readthedocs.io/en/stable/api.html#tenacity.retry.retry_if_exception_type

Reply:
Ahh, fixed that and fixed up the tests. Thanks for the review.